pkg/registry/core/service/storage/transaction.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
This should be generalized. I am sure we have we have objects out there that have similar semantics. even if we don't we can have that now.. we will eventually will.

Reply:
yeah, I'm following the "start local" mindset.  If we have others that are similar we can move this up and out LATER